documentation/staging/content/userguide/managing-domains/model-in-image/auxiliary-images.md

```diff
@@ -376,33 +241,29 @@ Model In Image model files, application archives, and the WDT installation files
 
        {{%expand "Click here to view the Dockerfile." %}}
        ```
-       # Copyright (c) 2021, Oracle and/or its affiliates.
+       # Copyright (c) 2021, 2022, Oracle and/or its affiliates.
        # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
-    
+       
        # This is a sample Dockerfile for supplying Model in Image model files
-       # and a WDT installation in a small separate auxiliary
+       # and a WDT install in a small separate auxiliary image
        # image. This is an alternative to supplying the files directly
        # in the domain resource `domain.spec.image` image.
-    
+              
        # AUXILIARY_IMAGE_PATH arg:
        #   Parent location for Model in Image model and WDT installation files.
-       #   Must match domain resource 'domain.spec.auxiliaryImageVolumes.mountPath'
-       #   For model-in-image, the following two domain resource attributes can
-       #   be a directory in the mount path:
-       #     1) 'domain.spec.configuration.model.modelHome'
-       #     2) 'domain.spec.configuration.model.wdtInstallHome'
```

Comment:
Add something like:

The default is '/auxiliary', which matches the parent directory in the default values for 'domain.spec.configuration.models.auxiliaryImageVolumes.sourceModelHome' and 'domain.spec.configuration.models.auxiliaryImageVolumes' respectively.